backend/lib/routes/index.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   '/cloudprofiles': require('./cloudprofiles'),
   '/seeds': require('./seeds'),
   '/shoots': require('./shoots'),
+  '/controllerregistrations': require('./controllerregistrations'),
```

Comment:
I think we should have to separate routes. 
* `controllerregistrations` or better `extensions` or `gardener-extensions`
* `networking-types` 
The networking-types is independent since we do not check any authorization here. The authorization check should not be less strict for a subresource. That is strange.

Reply:
I agree in that case I need to introduce a new route.
Regrading renaming - I don't like to call it extensions, I know we talk about extensions when a referring to them but as we are more or less returning the `controllerregistrations` resource I think we should keep the name.

Reply:
done, split into two routes:
`/gardener-extensions`
`/networking-types`